src/services/ruleEngine/TemplateRenderer.ts

```
@@ -0,0 +1,403 @@
+      case 'unemployment_days_remaining':
+        return String(context.calculated.unemploymentDaysRemaining || 'N/A');
+        
+      case 'time_remaining_friendly':
+        return context.calculated.timeRemaining || 'Unknown';
+        
+      case 'urgency_indicator':
+        return this.getUrgencyIndicator(context.calculated.urgencyLevel);
+        
+      default:
+        // Allow upstream (RuleEngine/DateCalculator) to supply arbitrary computed values
+        const v = (context.calculated as any)?.[calculation];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)